test/modelnode.jl

```diff
@@ -338,3 +338,34 @@ end
         end
     end
 end
+
+gradients_match(grads1::Flux.Zygote.Grads, grads2::Flux.Zygote.Grads) = grads1.params == grads2.params && gradients_match(grads1.grads, grads2.grads)
+gradients_match(grads1::Flux.Zygote.Grads, grads2::Flux.Zygote.Grads) = grads1.params == grads2.params && gradients_match(grads1.grads, grads2.grads)
+gradients_match(grads1::IdDict, grads2::IdDict) = length(grads1) == length(grads2) && all(kv1 == kv2 for (kv1, kv2) in zip(grads1, grads2))
+
+@testset "testing simple named tuple model with reduce catobs in gradient" begin
+    layerbuilder(k) = Flux.Dense(k, 2, NNlib.relu)
```

Comment:
Why not just "`Dense`" and "`relu`"?

Reply:
ah,y es, thanks, I forgot they are exposed